base/applications/regedit/main.c

```diff
@@ -97,8 +98,9 @@ BOOL InitInstance(HINSTANCE hInstance, int nCmdShow)
     hPopupMenus = LoadMenuW(hInstance, MAKEINTRESOURCEW(IDR_POPUP_MENUS));
 
     /* Initialize the Windows Common Controls DLL */
-    // TODO: Replace this call by InitCommonControlsEx(_something_)
-    InitCommonControls();
+    icce.dwSize = sizeof(icce);
+    icce.dwICC = 0xFFFF; /* Windows does this! */
```

Comment:
Shouldn't that be `ICC_WIN95_CLASSES`?

Reply:
Windows uses this value (`0xFFFF`) as above images.

Reply:
I seriously have allergies when I see magic values like these, specially with no proper description or meaning about these whatsoever. Please do some research on how is this value treated and what Windows does with it, having only a `Windows does this!` comment is pretty vague.

Reply:
This `0xFFFF` value would be the combination of:
```txt
ICC_WIN95_CLASSES 0x000000FF
ICC_DATE_CLASSES 0x00000100
ICC_USEREX_CLASSES 0x00000200
ICC_COOL_CLASSES 0x00000400
ICC_INTERNET_CLASSES 0x00000800
ICC_PAGESCROLLER_CLASS 0x00001000
ICC_NATIVEFNTCTL_CLASS 0x00002000
ICC_STANDARD_CLASSES 0x00004000
ICC_LINK_CLASS 0x00008000
```

Reply:
`ICC_WIN95_CLASSES | ICC_USEREX_CLASSES | ICC_LINK_CLASS` might be good enough. `ICC_USEREX_CLASSES` is for comboboxex controls. `ICC_LINK_CLASS` is for hyperlink controls. We can ignore `ICC_STANDARD_CLASSES` flag as next article: https://stackoverflow.com/questions/33312287/is-a-button-a-common-control

Reply:
Now optimized.